src/objects.js

```diff
@@ -1,41 +1,54 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name,
+    age,
+  }
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65 ) {
+    return true;
+  } else {
+     return false; 
+  }
 };
 
 const getAges = people => {
-  // your code here
+  return people.map (people => people.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find (x => x.name === name );
```

Comment:
Whitespace shouldnt be here
```suggestion
  return people.find(x => x.name === name );
```